src/jit/lsra.cpp

```diff
@@ -3374,23 +3375,40 @@ bool LinearScan::isRegInUse(RegRecord* regRec, RefPosition* refPosition)
             // (Or, in the case of ARM, the other half of a double is either active or has an active recentRefPosition).
```

Comment:
Do you still need this comment here, as it is now mentioned below...

Reply:
No, I'll remove that part of the comment.